template/Makefile

```diff
@@ -45,6 +45,11 @@ dev: ## Install development dependencies
 	npm install
 	opam install --deps-only --with-test --with-doc -y .
 
+.PHONY: create_switch
+create_switch:
+  # Create a local opam switch
```

Comment:
I think the comment needs to be in the same line as the name of the command for the regex to catch it and display the doc in `make help`:

```
.PHONY: create_switch
create_switch: ## Create a local opam switch
  ...
```